tests/templates/test_template_inheritance.py

```diff
@@ -401,5 +506,57 @@ def test_create_network_with_child_template(self, client):
         })
 
         new_network = client.add_network(network)
+        #Now do a get network to verify the network's there
+        requested_network = client.get_network(new_network.id)
+
+        assert requested_network.nodes[0].types[0].child_template_id == child_template_j.id
+
+        #Make sure that the correct attributes have been added from the child (inherihted
+        #from the parent)
+        assert len(requested_network.attributes) == len(parent_network_type.typeattrs)
+
+
+    def test_delete_parent_type(self, client):
+        """
+            Test to ensure that when you delete a parent type, its child types
+            are also deleted (or not depoending on the 'force' flag)
```

Comment:
Does the "force flag" in `delete_templatetype` referred to here mean that function's `force` or `delete_children` kwarg?  L541-542 below uses both terms.

Also, see the suggestion to `__init__.py` above regarding `force` vs `delete_resourcetypes`.